src/main/java/com/suse/salt/netapi/calls/LocalCall.java

```diff
@@ -83,6 +102,9 @@ public LocalCall(String functionName, Optional<List<?>> arg,
         arg.ifPresent(arg -> payload.put("arg", arg));
         kwarg.ifPresent(kwarg -> payload.put("kwarg", kwarg));
         metadata.ifPresent(m -> payload.put("metadata", m));
+        timeout.ifPresent(timeout -> payload.put("timeout", timeout));
```

Comment:
Well, these two parameters are now handled inside `salt.client.LocalClient` class of Salt, which is the class that handles all `local_*` clients.

The `local`, `local_async` clients will work as expected, but looking now at `local_batch` in the Salt side, it seems that kwargs won't be propagated to the Batch object so I'll expect that timeouts parameters won't work on this case. I'll fix this on Salt ASAP. Thanks! :+1: